rest-service/manager_rest/manager_exceptions.py

```diff
@@ -122,6 +122,11 @@ class NoTokenGeneratorError(ManagerException):
     status_code = 401
 
 
+class InvalidFernetTokenFormatError(ManagerException):
+    error_code = 'invalid_fernet_token_format_error'
+    status_code = 400
```

Comment:
Sorry, should've noticed this before: This is a server side error, so it should be a 500.